apis/system/tests/test_write_seurat_read_anndata.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+import tiledbsoma as soma
+from tiledbsoma import io
+
+from .common import (
+    TestWritePythonReadR,
+    embed_python_list_into_R_code,
+)
+
+
+class TestSeuratSOMAAnndata(TestWritePythonReadR):
+    """
+    This test will perform a Seurat->SOMA->Anndata conversion and verify that the final artifact (Anndata)
+    has the same properties of the initial one (Seurat)
+    """
+
+    @pytest.fixture(scope="class")
+    def anndata_from_seurat(self):
+        base_script = f"""
+        library("tiledbsoma")
+        library("SeuratObject")
+
+        seuratObject <- readRDS("apis/system/tests/data/pbmc-small.rds")
+        write_soma(seuratObject, "{self.uri}")
+        """
```

Comment:
Why are we bundling `pbmc_small` in this repo instead of using the copy provided by SeuratObject?

```R
library(SeuratObject)

data("pbmc_small")
write_soma(pbmc_small, "{self.uri}")
```

Reply:
Nice! I didn't know it existed. Changed.